### DISCUSSION TITLE
Information architecture/well-known URLs and variability

### DISCUSSION THREAD

**Discussion by matatk** [w3c/personalization-semantics#252] [Ideas]
@abhina-kumar points out that:

> In some SaaS applications, the URLs at time contain tenant-id (they use it to identify the tenant)?

I would like to understand this issue more. Is there a definition of "tenant-id" somewhere that we could refer to? I'd like to understand why a variable ID would be required as part of a URL.

My current sense is that this is likely to be unusual (relative to the size of the web), but I'd like to understand why it occurs - what is the benefit of using "variable URLs" to the app?

**Comment by abhina-kumar** [comment]
> @abhina-kumar points out that:
> 
> > In some SaaS applications, the URLs at time contain tenant-id (they use it to identify the tenant)?
> 
> I would like to understand this issue more. Is there a definition of "tenant-id" somewhere that we could refer to? I'd like to understand why a variable ID would be required as part of a URL.
> 
> My current sense is that this is likely to be unusual (relative to the size of the web), but I'd like to understand why it occurs - what is the benefit of using "variable URLs" to the app?

Hi @matatk,

1) Generally speaking clients are referred to as “tenants” in SaaS applications.
2) There are many ways to identify tenants. One of them is using "URL-based identification".
You may find more details in this [link](https://blog.ramdoot.in/introducing-multitenancy-upgrade-your-web-application-acb47ab90c30).
